bin/copy_shared_libs.sh

```diff
@@ -29,18 +30,31 @@ then
 else  
   if [ -x  ./flow123d ]
   then
+    # detect shared libs
     loader=`ldd flow123d | grep "/ld" | sed 's/^\t*\(\/.*\) (.*)$/\1/'`
-    libs_non_system=`ldd flow123d | grep " => /" | grep -v 'libc\.\|libdl\.\|libgcc_s\.\|libgfortran\.\|libm\.\|libstdc++\.\|librt\.' | sed 's/^.* => \(\/.*\) (.*)$/\1/'`
-    libs_system=`ldd flow123d | grep " => /" | grep 'libc\.\|libdl\.\|libgcc_s\.\|libgfortran\.\|libm\.\|libstdc++\.\|librt\.' | sed 's/^.* => \(\/.*\) (.*)$/\1/'`
-    libs_system="${libs_system} ${loader}"
+    libs=`ldd flow123d | grep -n "^/" | sed 's/^.* => \(\/.*\) (.*)$/\1/'`
+    libs="${libs}"
     
+    # copy shared libs
     echo "Copy flow123d shared libraries ..."
-    echo $libs_non_system
-    echo $libs_system
     if [ ! -d ../lib ]; then mkdir ../lib; fi
-    if [ ! -d ../lib/system ]; then mkdir ../lib/system; fi
-    cp $libs_non_system ../lib
-    cp $libs_system ../lib/system
+    cp ${libs} ../lib
+    
+    # manually copy and rename file to bin dir so cmake can make it executable
+    cp ${loader} ./ld-linux-loader.so
+    chmod +x "./ld-linux-loader.so"
+    
+    # now we alter wrapper file located in bin/flow123d_wrapper.sh
+    wrapper="flow123d_wrapper.sh"
+    echo "Generating wrapper file '$wrapper'"
+    echo "#!/bin/bash" > $wrapper
+    echo "DIR=\"\$( cd \"\$( dirname \"\${BASH_SOURCE[0]}\" )\" && pwd )\"" >> $wrapper
```

Comment:
1. Rather use here document for the script in order to avoid some escaping and improve readability. 
2. I do not understand meaning of this line, the part before pwd.